oidc-service-impl/src/main/java/io/mosip/esignet/services/ConsentHelperService.java

```diff
@@ -86,6 +86,32 @@ public void processConsent(OIDCTransaction transaction, boolean linked) {
         }
     }
 
+    public ConsentAction getConsentAction(OIDCTransaction transaction,boolean linked) {
```

Comment:
what is the reason for duplicating "processConsent" method? Why not reuse the existing method?

Reply:
removed the getConsentAction now using the ConsentAction that is stroed in authenticated transaction